Boardfiles/nucleol552zeq/Core/Src/i2c.c

```diff
@@ -82,7 +82,7 @@ void MX_I2C2_Init(void)
 
   /* USER CODE END I2C2_Init 1 */
   hi2c2.Instance = I2C2;
-  hi2c2.Init.Timing = 0x40505681;
+  hi2c2.Init.Timing = 0x20505885;

```

Comment:
Why did we need to change this timing? Or was it just a i2c configuration changed and this was part of the auto-generated files?

Reply:
I was using PLL clock for the SDMMC peripheral, and I changed the PLL freq to be slow enough for the SD card. I changed this so I'm not using PLL anymore and instead Clock48 with a clock divider factor of 8 for SDMMC peripheral.